spark/spark-temperature-analysis/src/main/scala/org/infinispan/quickstart/spark/TemperatureAnalysis.scala

```diff
@@ -62,7 +64,7 @@ object TemperatureAnalysis {
       val configIn = new Properties
       configIn.put("infinispan.rdd.cacheName", inputDataGridCache)
       configIn.put("infinispan.client.hotrod.server_list", dataGridServer)
-      val ispnStream = new InfinispanInputDStream[String, Double](ssc, StorageLevel.MEMORY_ONLY, configIn)
+      val ispnStream = new InfinispanInputDStream[String, Double](ssc, StorageLevel.MEMORY_ONLY, org.infinispan.spark.config.ConnectorConfiguration(configIn.asInstanceOf[java.util.Map[String, String]].asScala.toMap))
```

Comment:
The configuration via ```Properties``` has been [replaced](http://blog.infinispan.org/2017/04/infinispan-spark-connector-05-released.html).
So this snippet is better written as:
```scala
import org.infinispan.spark.config.ConnectorConfiguration;
...
val configIn = new ConnectorConfiguration().setCacheName("inputDataGridCache").setServerList(dataGridServer)
val ispnStream = new InfinispanInputDStream[String, Double](ssc, StorageLevel.MEMORY_ONLY, configIn) 
```